components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -19,11 +19,14 @@
 
 package integration.chown;
 
+import static omero.rtypes.rstring;
```

Comment:
Overtook the static import from https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroJava/test/integration/delete/AnnotationDeleteTest.java#L9